distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -44,25 +47,14 @@
      *
      * @return the minimum headers required for sending a chunked data response
      */
-    private static Action getHeaders() {
+    public static Action getChunkedResponseHeaders() {
```

Comment:
Do `getChunkedResponseHeaders` and `getRangeSupportHeaders` really have to be `public`? If so, could we add them to the builder instead?